packages/commonwealth/client/scripts/views/SublayoutBanners.tsx

```diff
@@ -37,13 +29,6 @@ export const SublayoutBanners = ({
       {banner && bannerStatus !== 'off' && (
         <CWMessageBanner bannerContent={banner} onClose={handleDismissBanner} />
       )}
-      {app.isLoggedIn() &&
-        ITokenAdapter.instanceOf(app.chain) &&
-        !app.user.activeAccount && (
-          <Old_CWBanner
-            bannerContent={`Link an address that holds ${chain.default_symbol} to participate in governance.`}
```

Comment:
Any reason this particular code block was removed?

Reply:
Originally removed because it was rendering even if a community didn't have governance (e.g. ERC20 or ERC721) community. Additionally, the chain object was null on the initial render in some cases leading to 'undefined' being shown in the banner (hadn't realized subsequent renders had chain defined). I just pushed a fix which should render the banner only for communities we support governance for and only when chain is defined.